### PR TITLE
Adding PyRe Slides to Presenation

### LINKED ISSUE #4: PyRE slide
Please add a slide (or two, but not more) describing :

- the current PyRE work you're working on
- how's it going to work
- when will it be ready
- what kinds of questions will it answer
- where is the repo
- who's involved

### PR DESCRIPTION
**Pull request by gtw2**
Adds 2 slides - the first contains how pyre works while the second contains details on "who, why, when"
Including flowchart from my paper for visual aid on the step by step separation process.

This would close #4.

### CODE REVIEW

**Comment by gtw2** [comment]
Per Gwen's suggestion - How PyRe works is now before "Current Work"

File: arfc-pres.tex [hunk 1]
```diff
@@ -94,7 +94,10 @@ \section{Cyder}
 \section{Demand Driven Cycamore Archetypes}
 \input{ddca}
 \section{PyRe}
+\subsection{PyRe progress and theory}
 \input{pyre}
+\subsection{About PyRe}
+\input{pyre2}
```
Comment (katyhuff) [inline arfc-pres.tex]: why are there two?

**Comment by katyhuff** [comment]
You can put subsection commands inside the pyre.tex file. Please submit just one file rather than pyre.tex and pyre2.tex.

File: pyre.tex [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+\begin{frame}
+\frametitle{PyRe: Cyclus (Py)ro (Re)processing Module}
+\begin{block}{Timeline:} 
+	The archetype will be functional with preliminary efficiencies early \textbf{August 2018}. \\
+	A more detailed PyRe will be completed for ANTPC in \textbf{September 2018}.
+\end{block}
+\begin{block}{Archetype Uses:} 
+	PyRe will answer the following questions
+	\begin{itemize}
+		\item What is the affect of introducing pyroprocessing plants in the fuel cycle?
```
Comment (katyhuff) [inline pyre.tex:39]: effect

File: arfc-pres.tex [hunk 3]
```diff
@@ -26,7 +26,7 @@
 \usepackage{booktabs} % nice rules for tables
 \usepackage{microtype} % if using PDF
 \usepackage{bigints}
-\usepackage{minted}
+%\usepackage{minted}
```
Comment (katyhuff) [inline arfc-pres.tex:29]: why no minted? I fully expect to use code examples in other slides. Will this break your slides?

Comment (gtw2) [reply arfc-pres.tex:29]: yeah that was to let mine compile

Comment (katyhuff) [reply arfc-pres.tex:29]: ok, well, I'm going to have to add it back .

**Comment by katyhuff** [comment]
Thank you!

**Pull request merged**